masschallenge/impact-api
# AC-6846: impact-api referencing the recommendation model causing failing builds

#### Changes introduced in [AC-6846](https://masschallenge.atlassian.net/browse/AC-6846)
- remove references to deprecated RecommendationTag

#### How to test
- a passing travis build should be good
- a passing local test run would be great as well
